src/main/java/emu/grasscutter/utils/ConfigContainer.java

```diff
@@ -176,7 +176,7 @@ public static class GameOptions {
         public boolean watchGachaConfig = false;
         public boolean enableShopItems = true;
         public boolean staminaUsage = true;
-        public boolean energyUsage = false;
+//        public boolean energyUsage = false;
```

Comment:
I don't get what the point or benefit of this change is supposed to be. On the contrary, I can only see this as a detriment. What does taking away server manager's ability to set a default for whether they want to use such a - highly experimental, by the way - feature improve?

Giving the ability to override this config with a command is fine. Persisting that is fine too (though you would have to devise a way to differentiate between nothing being set and the feature being turned off). But taking away the configurability is not fine.